_editorial_policies/01_bylaws.md

```diff
@@ -0,0 +1,149 @@
+
+   1. If a conflict of interest in a reviewer is found after acceptance, the associate editor will evaluate whether any bias may have affected the acceptance. If the associate editor feels they may be bias, a committee of three associate editors selected by the Section Lead Editor will investigate whether any bias or fraud was intended, and will follow procedures to determine whether admission was fraudulent as outlined by the bylaws on fraud.
+
+   1. No Section Lead Editor shall assign an Associate Editor if they
+have a conflict of interest with a paper. No Associate Editor may
+manage a paper if they have a conflict of interest, and shall notify
+the Section Lead Editor if they do have a conflict.
+
+   1. If a Section Lead Editor has a conflict of interest, the
+assignments to Associate Editors will be made by another Section Lead
+Editor.  The order will be cyclic; if Lead Section Editor A has a
+conflict, it will got to Lead Section Editor B, unless they also have
+a conflict, in which case it will go to C, and so on, returning to A
+when the sections are exhausted. In the case that all of the lead
+editors have conflicts, the Managing Editors will jointly pick an
+Associate Editor who will then assign another Associate Editor to
+manage the manuscript.
+      1. Comparison of Molecular Simulation Package Editor
+      1. Tutorials Editor
+      1. Best Practices Editor
+      1. Review Editor
+      1. Lessons Learned Editor
+      1. continuing with editors of other sections, in order they are chronologically added to the journal.
+
+1. Fraud and Plagiarism
+
+    1. Definitions:
+
+       1. 'Plagiarism' is defined as including any clearly recognizable
+amount of text from a previous publication without attribution.
+
+       1. 'Fraud' is defined as any false representation of the circumstances of the paper or the data contained in the paper. It could include fraudulent data, or fraudulent reviews.
+
+    1. If there is a suspicious or accusation of fraud or plagiarism, a
+       committee of three members of the editorial board will be chosen by
+       the managing editors to investigate.  If this committee votes
+       unanimously that fraud or plagiarism occurred, then the paper will
+       be removed from the journal, and the author will not be allowed to
+       submit to LiveCoMS for three years.
+
+1. Review Process
+
+    1. Review criteria that editors use shall be posted on the author
+       instructions section of the website. These review criteria will be
+       revised for clarity periodically in response to feedback, with the
+       approval of the Section Lead Editor.
+
+    2. Section Lead Editors make the final decision about reviews in
+       consultation with the associate editor assigned to the
+       paper. In case of an appeal on the decision, the Section Lead
+       Editor will select another associate editor to review all
+       materials associated with the decision. All authors must agree
```

Comment:
The "all authors must agree..." sentence seems out of place; does this belong earlier or later? This concerns how appeals are initiated, so it seems out of place in the midst of the discussion of how appeals are handled once initiated.

Reply:
All corresponding authors?  For that matter, did we somewhere address the issue of corresponding authors?

Reply:
Is there a better place?  It's right after the first mention of appeals. 

I was thinking all authors, but is there an argument for just corresponding authors?

Reply:
Maybe instead of "In case of an appeal on the decision..." you should say, "In some cases, authors may appeal an editorial decision, but initiating an appeal requires the consent of all authors. In the case of an appeal, the Section Lead Editor..."

Reply:
OK, changed to that.